src/remotes/auth0/index.js

```diff
@@ -4,10 +4,13 @@ import { delegateToSchema } from '@graphql-tools/delegate';
 import { makeExecutableSchema } from '@graphql-tools/schema';
 import { TransformQuery } from '@graphql-tools/wrap';
 import { Kind } from 'graphql';
-import { scopes, requireScope } from '../../auth';
+import { scopes, requireScope, hasScope } from '../../auth';
```

Comment:
Perhaps instead of `hasScope` and throwing errors manually every time, the code would be cleaner if we added a method like:

```js
export const requireAnyOfScopes = (ctx, scopes) => {
  if (!scopes.reduce((accum, scope) => accum || hasScope(ctx, scope), false)) {
    throw new Error(`Your request requires one of the scopes: ${scopes.join(', ')}.`);
  }
  return true;
};
```